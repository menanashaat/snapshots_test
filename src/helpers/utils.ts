export function formatNumber(number: number) {
  return new Intl.NumberFormat('de-Da').format(number) // Exmaple 123456 ====> 123,456
}
