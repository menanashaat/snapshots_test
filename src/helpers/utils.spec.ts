// formatNumber.test.js
import { describe, it, expect } from 'vitest'
import { formatNumber } from './utils'

describe('formatNumber function', () => {
  it('should format number in de-DE locale correctly', () => {
    const result = formatNumber(123456.78)
    expect(result).toMatchSnapshot() // Snapshot saved with output "123.456,78"
  })
})
