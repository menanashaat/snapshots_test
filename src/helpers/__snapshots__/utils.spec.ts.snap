// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`formatNumber function > should format number in de-DE locale correctly 1`] = `"123.456,78"`;
